include/onnxruntime/core/session/onnxruntime_cxx_inline.h

```diff
@@ -362,10 +362,20 @@ inline RunOptions& RunOptions::UnsetTerminate() {
   return *this;
 }
 
+inline SessionOptions::SessionOptions(std::nullptr_t) {
```

Comment:
no - this constructor is for people that want this to be empty.   a null ptr on purpose.   they plan on attaching a container ptr later.  we don't want to construct anything here.

Reply:
So that means we will miss detect the usage of C++ as C(default) if people use the constructor.